create es table failed

<!-- (At least include the following, feel free to add if you have more content) -->

### Steps to reproduce the behavior (Required)
java.lang.NoClassDefFoundError: org/codehaus/jackson/map/ObjectMapper
        at com.starrocks.connector.elasticsearch.EsRestClient.<init>(EsRestClient.java:84) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.elasticsearch.EsRestClient.<init>(EsRestClient.java:103) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.elasticsearch.EsRepository.registerTable(EsRepository.java:77) ~[starrocks-fe.jar:?]
        at com.starrocks.catalog.EsTable.onReload(EsTable.java:561) ~[starrocks-fe.jar:?]
        at java.util.ArrayList.forEach(ArrayList.java:1259) ~[?:1.8.0_292]
        at com.starrocks.server.LocalMetastore.load(LocalMetastore.java:4667) ~[starrocks-fe.jar:?]
        at com.starrocks.server.GlobalStateMgr.loadImage(GlobalStateMgr.java:1533) ~[starrocks-fe.jar:?]
        at com.starrocks.server.GlobalStateMgr.initialize(GlobalStateMgr.java:1094) ~[starrocks-fe.jar:?]
        at com.starrocks.StarRocksFE.start(StarRocksFE.java:133) ~[starrocks-fe.jar:?]
        at com.starrocks.StarRocksFE.main(StarRocksFE.java:84) ~[starrocks-fe.jar:?]
Caused by: java.lang.ClassNotFoundException: org.codehaus.jackson.map.ObjectMapper
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_292]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_292]
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_292]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_292]
        ... 10 more

### Expected behavior (Required)

### Real behavior (Required)

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

--- a/fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/EsRestClient.java
+++ b/fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/EsRestClient.java
@@ -34,7 +34,10 @@
 
 package com.starrocks.connector.elasticsearch;
 
+import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.collect.ImmutableList;
 import com.starrocks.connector.exception.StarRocksConnectorException;
 import okhttp3.Credentials;
@@ -45,10 +48,6 @@ import org.apache.http.HttpHeaders;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.util.Strings;
-import org.codehaus.jackson.JsonParser;
-import org.codehaus.jackson.map.DeserializationConfig;
-import org.codehaus.jackson.map.ObjectMapper;
-import org.codehaus.jackson.map.SerializationConfig;
 
 import java.io.IOException;
 import java.security.SecureRandom;
@@ -82,8 +81,7 @@ public class EsRestClient {
 
     {
         mapper = new ObjectMapper();
-        mapper.configure(DeserializationConfig.Feature.USE_ANNOTATIONS, false);
-        mapper.configure(SerializationConfig.Feature.USE_ANNOTATIONS, false);
+        mapper.configure(MapperFeature.USE_ANNOTATIONS, false);
     }
 
     private static final OkHttpClient NETWORK_CLIENT = new OkHttpClient.Builder()

--- a/fe/fe-core/src/main/java/com/starrocks/http/rest/GetLogFileAction.java
+++ b/fe/fe-core/src/main/java/com/starrocks/http/rest/GetLogFileAction.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.http.rest;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Strings;
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
@@ -44,7 +45,6 @@ import com.starrocks.http.BaseResponse;
 import com.starrocks.http.IllegalArgException;
 import io.netty.handler.codec.http.HttpMethod;
 import io.netty.handler.codec.http.HttpResponseStatus;
-import org.codehaus.jackson.map.ObjectMapper;
 
 import java.io.File;
 import java.util.Map;

--- a/fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseAction.java
+++ b/fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseAction.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.http.rest;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.starrocks.common.DdlException;
 import com.starrocks.common.Pair;
 import com.starrocks.common.util.UUIDUtil;
@@ -50,7 +51,6 @@ import io.netty.handler.codec.http.HttpHeaderNames;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.codehaus.jackson.map.ObjectMapper;
 
 import java.net.URI;
 import java.net.URISyntaxException;

--- a/fe/fe-core/src/main/java/com/starrocks/http/rest/TableQueryPlanAction.java
+++ b/fe/fe-core/src/main/java/com/starrocks/http/rest/TableQueryPlanAction.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.http.rest;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Strings;
 import com.starrocks.analysis.TableName;
 import com.starrocks.catalog.Database;
@@ -72,7 +73,6 @@ import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.thrift.TException;
 import org.apache.thrift.TSerializer;
-import org.codehaus.jackson.map.ObjectMapper;
 import org.json.JSONException;
 import org.json.JSONObject;
 

--- a/fe/fe-core/src/main/java/com/starrocks/http/rest/TableRowCountAction.java
+++ b/fe/fe-core/src/main/java/com/starrocks/http/rest/TableRowCountAction.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.http.rest;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Strings;
 import com.starrocks.catalog.Database;
 import com.starrocks.catalog.OlapTable;
@@ -49,7 +50,6 @@ import com.starrocks.qe.ConnectContext;
 import com.starrocks.server.GlobalStateMgr;
 import io.netty.handler.codec.http.HttpMethod;
 import io.netty.handler.codec.http.HttpResponseStatus;
-import org.codehaus.jackson.map.ObjectMapper;
 
 import java.util.HashMap;
 import java.util.Map;

--- a/fe/fe-core/src/main/java/com/starrocks/http/rest/TableSchemaAction.java
+++ b/fe/fe-core/src/main/java/com/starrocks/http/rest/TableSchemaAction.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.http.rest;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Strings;
 import com.starrocks.catalog.Column;
 import com.starrocks.catalog.Database;
@@ -53,7 +54,6 @@ import com.starrocks.qe.ConnectContext;
 import com.starrocks.server.GlobalStateMgr;
 import io.netty.handler.codec.http.HttpMethod;
 import io.netty.handler.codec.http.HttpResponseStatus;
-import org.codehaus.jackson.map.ObjectMapper;
 
 import java.util.ArrayList;
 import java.util.HashMap;

--- a/fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java
+++ b/fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java
@@ -34,6 +34,7 @@
 
 package com.starrocks.server;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Strings;
 import com.google.common.collect.Lists;
@@ -79,7 +80,6 @@ import com.starrocks.thrift.TUpdateResourceUsageRequest;
 import com.starrocks.thrift.TUpdateResourceUsageResponse;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.codehaus.jackson.map.ObjectMapper;
 
 import java.io.DataInputStream;
 import java.io.DataOutputStream;

--- a/fe/fe-core/src/test/java/com/starrocks/connector/elasticsearch/PartitionPhaseTest.java
+++ b/fe/fe-core/src/test/java/com/starrocks/connector/elasticsearch/PartitionPhaseTest.java
@@ -34,14 +34,14 @@
 
 package com.starrocks.connector.elasticsearch;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.starrocks.catalog.Column;
 import com.starrocks.catalog.EsTable;
 import com.starrocks.catalog.Type;
 import com.starrocks.common.ExceptionChecker;
 import mockit.Expectations;
 import mockit.Injectable;
-import org.codehaus.jackson.JsonParser;
-import org.codehaus.jackson.map.ObjectMapper;
 import org.junit.Test;
 
 import java.util.ArrayList;

